Homework-3/script.js

```diff
@@ -0,0 +1,50 @@
+`use strict`;
+// 1
+const user = {};
+
+user.name = `John`;
+user.surname = `Smith`;
+
+user.name = `Pete`;
+
+delete user.name;
+
+// 2
+function isEmpty(obj) {
+	for (const key in obj) {
+		return "false";

```

Comment:
false as a string? Are you out of your mind?
return false. just false. without quotes